katello.gemspec

```
@@ -66,14 +66,6 @@ Gem::Specification.new do |gem|
   gem.add_dependency "angular-rails-templates", "~> 1.1.0"
   gem.add_development_dependency "uglifier"
 
-  # Testing
-  gem.add_development_dependency "factory_bot_rails", "~> 4.5"
-  gem.add_development_dependency "minitest-tags"
-  gem.add_development_dependency "mocha"
-  gem.add_development_dependency "vcr", "< 4.0.0"
-  gem.add_development_dependency "webmock"
-  gem.add_development_dependency "rubocop-checkstyle_formatter"
-  gem.add_development_dependency "simplecov"
-  gem.add_development_dependency "simplecov-rcov"
-  gem.add_development_dependency "robottelo_reporter"
+  # Rubocop
+  gem.add_development_dependency 'theforeman-rubocop', '~> 0.0.6'
```

We have 0.1.2 as well, which is actually new enough for GH annotations.
```suggestion
  gem.add_development_dependency 'theforeman-rubocop', '~> 0.1.0'
```

---

yeah, it was just a test to pull something in, and then I erred on the side of "what's in foreman today"